be/src/vec/functions/round.h

```diff
@@ -217,75 +230,30 @@ class BaseFloatRoundingComputation<Float64> {
     static VectorType divide(VectorType val, VectorType scale) { return _mm_div_pd(val, scale); }
     template <RoundingMode mode>
     static VectorType apply(VectorType val) {
-        return _mm_round_pd(val, int(mode));
+        if constexpr (mode != RoundingMode::Round) {
+            return _mm_round_pd(val, int(mode));
```

Comment:
we could use a marco to check the SSE4.1 enable state. and use the most proper simd instructions with `_mm` or `_mm256`. u can see the similar usage in our code.